lib/stellar_core_commander/process.rb

```diff
@@ -3,15 +3,25 @@ module StellarCoreCommander
   class Process
     include Contracts
 
+    attr_reader :transactor
     attr_reader :working_dir
     attr_reader :base_port
     attr_reader :identity
     attr_reader :server
+    attr_reader :unverified
+    attr_writer :unverified
```

Comment:
Most ruby code I see collapses the two lines above into `attr_accessor :unverified`